### PR TITLE
#fix  Remove travis

### PR DESCRIPTION
**Pull request by duker33**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -73,6 +73,19 @@ pipeline:
     when:
       branch: master
 
+  lint:
+    image: docker/compose:1.17.1
+    commands:
+      - cd docker
+      - docker-compose run --rm stb-lint
+      - docker-compose run --rm stb-pdd
```
Comment (dveselov) [inline .drone.yml]: `se-lint` и `se-pdd`

File: .drone.yml [hunk 2]
```diff
@@ -69,10 +69,26 @@ pipeline:
       - docker-compose -f docker-compose-production.yml build --no-cache se-python
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - /root/prog/shopelectro/docker/.env:/drone/src/github.com/fidals/shopelectro/commit/${DRONE_COMMIT_SHA}/docker/.env
+      # in case if "Push Hooks" is enabled in Drone settings GUI
+      - /root/prog/stroyprombeton/docker/.env:/drone/src/github.com/fidals/stroyprombeton/commit/${DRONE_COMMIT_SHA}/docker/.env
+      # in case if "Pull Request Hooks" is enabled in Drone settings GUI
+      - /root/prog/stroyprombeton/docker/.env:/drone/src/github.com/fidals/stroyprombeton/pull/${DRONE_PULL_REQUEST}/docker/.env
```
Comment (dveselov) [inline .drone.yml]: Случайно скопипастилось `stroyprombeton` -> `shopelectro` :)

Comment (duker33) [reply .drone.yml]: мдя, сорян

**Comment by 0pdd** [comment]
@duker33 the puzzle [#732](https://github.com/fidals/shopelectro/issues/732) is still not solved.

**Pull request merged**